keyboards/dygma/raise/readme.md

```diff
@@ -0,0 +1,57 @@
+# Raise
+
+<a href="http://www.dygma.com">
+  <img src="https://cdn.shopify.com/s/files/1/0374/9448/9228/files/Dygma-Raise-Hero2.png" width="600">
+</a>
+
+* Keyboard Maintainer: [ibash](https://github.com/ibash)
+* Hardware Supported: [F411 Blackpill](https://github.com/WeActTC/MiniSTM32F4x1) with the [Dygma Raise](http://www.dygma.com)
+* Hardware Availability: See below
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make dygma/raise:ansi
+
+Flashing example for this keyboard:
+
+    make dygma/raise:ansi:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Hardware
+
+QMK does not currently support the [Dygma Neuron](https://dygma.com/products/neuron-black) because QMK does not support SAMD21 which the Neuron has.
+Instead this code uses a [STM32 Blackpill](https://github.com/WeActTC/MiniSTM32F4x1) to create a Neuron replacement.
+
+## Required Hardware
+
+1. [STM32 Blackpill from WeAct Studio](https://github.com/WeActTC/MiniSTM32F4x1) (via [aliexpress](https://www.aliexpress.com/item/1005001456186625.html) or [adafruit](https://www.adafruit.com/product/4877)).
```

Comment:
As you mention here, the project should really be relocated to something like `keyboards/handwired/dygma/raise`.